applications/nrf_desktop/doc/board.rst

```diff
@@ -0,0 +1,39 @@
+.. _board:
+
+Board module
+############
+
+The ``board`` module is used to assure proper state of GPIO pins which are connected to electrical components and not configured by other modules in the application.
+
+Module Events
+*************
+
++----------------+-------------+--------------+-----------------+------------------+
+| Source Module  | Input Event | This Module  | Output Event    | Sink Module      |
++================+=============+==============+=================+==================+
+|                |             | ``board``    |                 |                  |
++----------------+-------------+--------------+-----------------+------------------+
+
+Configuration
+*************
+
+The module uses the Zephyr :ref:`zephyr:gpio` driver to set pin state.
+For this reason, you should set :option:`CONFIG_GPIO` option.
+
+For every configuration you must define ``port_state_def.h`` file (in board-specific directory in the application configuration folder).
+
+Mentioned file defines port states by the following arrays:
```

Comment:
```suggestion
The ``port_state_def.h`` file defines the port states by the following arrays:
```
What do you mean with `by the following arrays`? Clarify in the text.

Reply:
ok